framework_tvb/tvb/tests/framework/core/entities/file/files_helper_test.py

```diff
@@ -155,57 +147,49 @@ def test_write_operation_metadata(self):
         self.files_helper.update_operation_metadata(self.PROJECT_NAME, "new_group_name", operation.id) 
         found_dict = XMLReader(expected_file).read_metadata()  
         assert "new_group_name" == found_dict['user_group']
-        
-    
-    def test_remove_dt_happy_flow(self):
+
+    def test_remove_dt_happy_flow(self, dummy_datatype_index_factory):
         """
         Happy flow for removing a file related to a DataType.
         """
-        folder_path = self.files_helper.get_project_folder(self.test_project, "42")
-        datatype = MappedType()
-        datatype.storage_path = folder_path
-        open(datatype.get_storage_file_path(), 'w') 
-        assert os.path.exists(datatype.get_storage_file_path()), "Test file was not created!"
-        self.files_helper.remove_datatype(datatype) 
-        assert not os.path.exists(datatype.get_storage_file_path()), "Test file was not deleted!"
-        
-        
-    def test_remove_dt_non_existent(self):
+        datatype = dummy_datatype_index_factory()
+        h5_path = h5.path_for_stored_index(datatype)
+        assert os.path.exists(h5_path), "Test file was not created!"
+        self.files_helper.remove_datatype_file(h5_path)
+        assert not os.path.exists(h5_path), "Test file was not deleted!"
+
+    def test_remove_dt_non_existent(self, dummy_datatype_index_factory):
         """
         Try to call remove on a dataType with no H5 file.
         Should work.
         """
-        folder_path = self.files_helper.get_project_folder(self.test_project, "42")
-        datatype = MappedType()
-        datatype.storage_path = folder_path
-        assert not os.path.exists(datatype.get_storage_file_path())
-        self.files_helper.remove_datatype(datatype)
-        
+        datatype = dummy_datatype_index_factory()
+        h5_path = h5.path_for_stored_index(datatype)
+        wrong_path = os.path.join(h5_path, "WRONG_PATH")
+        assert not os.path.exists(wrong_path)
+        self.files_helper.remove_datatype_file(wrong_path)
 
-    def test_move_datatype(self):
+    def test_move_datatype(self, dummy_datatype_index_factory):
         """
         Make sure associated H5 file is moved to a correct new location.
         """
         folder_path = self.files_helper.get_project_folder(self.test_project, "42")
-        datatype = MappedType()
+        datatype = dummy_datatype_index_factory()
         datatype.storage_path = folder_path
-        open(datatype.get_storage_file_path(), 'w') 
-        assert os.path.exists(datatype.get_storage_file_path()), "Test file was not created!"
+        assert os.path.exists(folder_path), "Test file was not created!"
```

Comment:
storage_path should not be used on datatype. Probably won't be necessary once the move method is fixed.